### PR TITLE
New change

### PR DESCRIPTION
**Pull request by mwildehahn**
another PR

### CODE REVIEW

File: updates.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 Sample updates
 
 Test link markup
```
Comment (mh-devel) [inline updates.md:3]: something here

**Review by mh-devel — APPROVED** [review]
i think this all looks good, one comment:

```
def something is good
```

also: `bunnies`

File: updates.md [hunk 2]
```diff
@@ -1,3 +1,47 @@
...
+
+Some other comment
+
+Sample updates
+
+Test link markup
+
+Some other comment
+
+Sample updates
```
Comment (mh-devel) [inline updates.md:19]: comment 1

File: updates.md [hunk 3]
```diff
@@ -1,3 +1,47 @@
...
+
+Test link markup
```
Comment (mh-devel) [inline updates.md:21]: comment 2

File: updates.md [hunk 4]
```diff
@@ -1,3 +1,47 @@
...
+
+Some other comment
+
+Sample updates
```
Comment (mh-devel) [inline updates.md:25]: comment 3

File: updates.md [hunk 5]
```diff
@@ -1,3 +1,47 @@
...
+
+Test link markup
```
Comment (mh-devel) [inline updates.md:27]: comment 4

File: updates.md [hunk 6]
```diff
@@ -1,3 +1,47 @@
...
+
+Some other comment
+
+Sample updates
+
```
Comment (mh-devel) [inline updates.md:32]: comment 5

File: updates.md [hunk 7]
```diff
@@ -1,3 +1,47 @@
...
+Test link markup
+
+Some other comment
+
+Sample updates
```
Comment (mh-devel) [inline updates.md:37]: comment 6

**Review by mh-devel — APPROVED** [review]
:shipit:

File: updates.md (hunk 6 above)
Comment (mwildehahn) [reply updates.md:32]: some other comment

File: updates.md (hunk 7 above)
Comment (mwildehahn) [reply updates.md:37]: some response

Comment (mh-devel) [reply updates.md:37]: MY RESPONSE TO YOU SIR

Comment (mwildehahn) [reply updates.md:37]: a couple responses

File: updates.md (hunk 6 above)
Comment (mwildehahn) [reply updates.md:32]: at once

File: updates.md (hunk 5 above)
Comment (mwildehahn) [reply updates.md:27]: here

File: updates.md (hunk 4 above)
Comment (mwildehahn) [reply updates.md:25]: also here

File: updates.md (hunk 3 above)
Comment (mwildehahn) [reply updates.md:21]: WHY!

Comment (mwildehahn) [reply updates.md:21]: These

File: updates.md (hunk 4 above)
Comment (mwildehahn) [reply updates.md:25]: Should

File: updates.md (hunk 5 above)
Comment (mwildehahn) [reply updates.md:27]: All

File: updates.md (hunk 6 above)
Comment (mwildehahn) [reply updates.md:32]: Rollup

File: updates.md (hunk 7 above)
Comment (mh-devel) [reply updates.md:37]: these

File: updates.md (hunk 6 above)
Comment (mh-devel) [reply updates.md:32]: should

File: updates.md (hunk 5 above)
Comment (mh-devel) [reply updates.md:27]: rollup

File: updates.md [hunk 8]
```diff
@@ -1,3 +1,47 @@
 Sample updates
 
 Test link markup
+
+Some other comment
```
Comment (mwildehahn) [inline updates.md:5]: something here

File: updates.md [hunk 9]
```diff
@@ -1,3 +1,47 @@
...
+
```
Comment (mwildehahn) [inline updates.md:6]: something else here

Comment (mwildehahn) [reply updates.md:6]: another comment

File: updates.md (hunk 8 above)
Comment (mwildehahn) [reply updates.md:5]: another comment

**Review by mwildehahn — COMMENTED** [review]
something here

```
with a code block
```

something else

File: updates.md [hunk 10]
```diff
@@ -1,3 +1,47 @@
 Sample updates
 
 Test link markup
+
```
Comment (mh-devel) [inline updates.md:4]: approving this

Comment (mh-devel) [reply updates.md:4]: so approved!

Comment (mh-devel) [reply updates.md:4]: the most approved!

**Review by mh-devel — APPROVED** [review]
Approved!

File: updates.md [hunk 11]
```diff
@@ -1,3 +1,47 @@
...
+Some other comment
+
+Sample updates
```
Comment (mh-devel) [inline updates.md:7]: Denied

Comment (mh-devel) [reply updates.md:7]: The most requested changes

Comment (mh-devel) [reply updates.md:7]: Make reviews great again

Comment (mh-devel) [reply updates.md:7]: it was funnier when you weren't actually president

**Review by mh-devel — CHANGES_REQUESTED** [review]
make some changes

File: updates.md (hunk 11 above)
Comment (mh-devel) [reply updates.md:7]: what happens now?

File: updates.md (hunk 9 above)
Comment (mwildehahn) [reply updates.md:6]: work now?

File: updates.md [hunk 12]
```diff
@@ -1,3 +1,47 @@
 Sample updates
 
 Test link markup
```
Comment (mwildehahn) [inline updates.md:3]: what position is this?

**Review by mwildehahn — COMMENTED** [review]
something

File: updates.md (hunk 4 above)
Comment (mwildehahn) [inline updates.md:25]: something here

Comment (mwildehahn) [reply updates.md:25]: another comment here

File: updates.md [hunk 13]
```diff
@@ -1,3 +1,47 @@
...
+
+Some other comment
+
+Sample updates
+
+Test link markup
+
+Some other comment
+
+Sample updates
```
Comment (mwildehahn) [inline updates.md:31]: something else here

**Review by mwildehahn — COMMENTED** [review]
overall comment

```
test

something
```

`here`

File: updates.md [hunk 14]
```diff
@@ -1,3 +1,47 @@
...
+
+Test link markup
```
Comment (mwildehahn) [inline updates.md:33]: new comment

File: updates.md (hunk 7 above)
Comment (mwildehahn) [inline updates.md:37]: another new comment

File: updates.md [hunk 15]
```diff
@@ -1,3 +1,47 @@
...
+
+Some other comment
```
Comment (mwildehahn) [inline updates.md:41]: another new comment

Comment (mwildehahn) [inline updates.md:41]: something else

Comment (mwildehahn) [reply updates.md:41]:
```
def something here

something else here
```

**Review by mwildehahn — COMMENTED** [review]
boop!

File: updates.md [hunk 16]
```diff
@@ -1,3 +1,47 @@
...
+
+Sample updates
+
```
Comment (mwildehahn) [inline updates.md:26]: first

Comment (mwildehahn) [reply updates.md:26]: second

File: updates.md (hunk 2 above)
Comment (mwildehahn) [inline updates.md:19]: third

Comment (mwildehahn) [reply updates.md:19]: fourth

File: updates.md [hunk 17]
```diff
@@ -1,3 +1,47 @@
...
+Test link markup
+
+Some other comment
```
Comment (mwildehahn) [inline updates.md:23]: fifth

Comment (mwildehahn) [reply updates.md:23]: sixth

**Review by mwildehahn — COMMENTED** [review]
beep boop

File: updates.md (hunk 7 above)
Comment (mwildehahn) [inline updates.md:37]: new comment

Comment (mwildehahn) [reply updates.md:37]: second comment

File: updates.md (hunk 2 above)
Comment (mwildehahn) [inline updates.md:19]: should be first

Comment (mwildehahn) [reply updates.md:19]: should be second

**Review by mwildehahn — COMMENTED** [review]
This looks awesome! 🚀

File: updates.md (hunk 4 above)
Comment (mwildehahn) [inline updates.md:25]: this looks good

Comment (mwildehahn) [reply updates.md:25]: another one here

File: updates.md [hunk 18]
```diff
@@ -1,3 +1,47 @@
...
+
```
Comment (mwildehahn) [inline updates.md:18]: something else i thought of here

Comment (mwildehahn) [reply updates.md:18]: more more more

Comment (mwildehahn) [reply updates.md:18]: stuffs

File: updates.md (hunk 17 above)
Comment (mwildehahn) [inline updates.md:23]: did you see this?!!?

File: updates.md (hunk 18 above)
Comment (mwildehahn) [reply updates.md:18]: this is cool too:

```
def something
```

something: https://github.com/mhahn/devel-github-integration/pull/2/files

File: updates.md (hunk 14 above)
Comment (mwildehahn) [inline updates.md:33]: not included

Comment (mwildehahn) [reply updates.md:33]: below fold

**Review by mwildehahn — COMMENTED** [review]
I'm liking this, lets 🚀 it!

File: updates.md (hunk 15 above)
Comment (mwildehahn) [inline updates.md:41]: new review

Comment (mwildehahn) [reply updates.md:41]: here

**Review by mwildehahn — COMMENTED** [review]
something

**Comment by mwildehahn** [comment]
something

File: updates.md (hunk 15 above)
Comment (mwildehahn) [reply updates.md:41]: something here

File: updates.md [hunk 19]
```diff
@@ -1,3 +1,47 @@
...
+Sample updates
+
+Test link markup
+
```
Comment (mwildehahn) [inline updates.md:34]: brand new

Comment (mwildehahn) [reply updates.md:34]: will get included

**Review by mwildehahn — COMMENTED** [review]
boo!

File: updates.md (hunk 4 above)
Comment (mwildehahn) [reply updates.md:25]: something here

**Comment by mwildehahn** [comment]
some comment

**Comment by mwildehahn** [comment]
something

File: updates.md (hunk 4 above)
Comment (mwildehahn) [reply updates.md:25]: New Review

**Review by mwildehahn — COMMENTED** [review]
👍

**Pull request merged**